Jenkinsfile

```diff
@@ -5,6 +5,7 @@ pipeline {
 			steps {
 				deleteDir()
 				checkout scm
+				rm -f /home/lisk/.lisky/config.lock
```

Comment:
I would rather remove the lock file in the `post` either in `failure` or in `always`.

Reply:
The problem would be in concurrent builds, that it could have been created and not yet deleted.

Reply:
True... why is the lock file located under `/home/lisk/` and not under `$WORKSPACE`?

Reply:
We have set it to `os.homedir()` - so this depends on Jenkins setup of lisky. In my example it would be in `/Users/tobias/.lisky/config.lock`